p256/tests/affine.rs

```diff
@@ -118,3 +118,14 @@ fn identity_encoding() {
             .is_identity()
     ))
 }
+
+#[test]
+fn noncompatible_is_none() {
+    let noncompactable_secret = [
+        175, 232, 180, 255, 91, 106, 124, 191, 224, 31, 177, 208, 236, 127, 191, 169, 201, 217,
+        75, 141, 184, 175, 120, 85, 171, 8, 54, 57, 33, 177, 83, 211,
+    ];
+    let secret = p256::SecretKey::from_be_bytes(&noncompactable_secret).unwrap();
```

Comment:
The latest release of `elliptic-curve` defines a specific endianness per-curve:

```suggestion
    let secret = p256::SecretKey::from_bytes(&noncompactable_secret).unwrap();
```